khub_1.py

```diff
@@ -0,0 +1,19 @@
+from selenium import webdriver
+import getpass
+#일단 이산수학만 지정해놓고 실행
+driver = webdriver.Chrome('C:\chromedriver.exe')
+driver.get('https://jbnu.khub.kr/')
+
+id = input("학번을 입력하세요 : ")
+pw = getpass.getpass("비밀번호를 입력하세요 : ")
+driver.find_element_by_name('username').send_keys(id)
```

Comment:
```suggestion
driver.find_element_by_name('username').send_keys(id)
driver.find_element_by_name('password').send_keys(pw)
```
name을 각각 'login', 'passwd'로 수정하니 실행되네용 확인부탁합니다~

Reply:
웅 그거 어제 제가 말해서 바꿨는데 미르가 아직 안올린거예용

Reply:
아하아 넵!!